### PR TITLE
Array Union

### PR DESCRIPTION
**Pull request by ArtemDomnikoff**

### CODE REVIEW

File: Array Union/1_Array_Union.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		if number in first:
+			delete_element(first,number)
+		elif number in second:
+			delete_element(second,number)
+		return []
+def delete_element(array,element):
+	array.remove(element)
+	return array
+result = []
+first = (input("Введите числа:")).split(",")
```
Comment (daria-gh) [inline Array Union/1_Array_Union.py]: English

File: Array Union/1_Array_Union.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+second = (input("Введите числа:")).split(",")
+while result == []:
```
Comment (daria-larionova) [inline Array Union/1_Array_Union.py]: why do we need this ```while result == []``` ? we could call ```array_union``` once and thats enough

File: Array Union/1_Array_Union.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+				number = int(number)
+				result.append(number)
+		for number in second:
+				number = int(number)
+				result.append(number)
+		return result
+	except ValueError:
+		print("Something went wrong.")
+		if number in first:
+			delete_element(first,number)
```
Comment (daria-gh) [inline Array Union/1_Array_Union.py:23]: will that work? ```first = [], second = [], return []```

**Review by daria-larionova — CHANGES_REQUESTED** [review]

File: Array Union/1_Array_Union.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+def array_union(first,second):
+	try:
+		result = []
+		for number in first:
+			if number not in second:
```
Comment (daria-gh) [inline Array Union/1_Array_Union.py]: ```if number not in second``` is standard python functionality.
let's change it to your own function to understand how it works
```if contains(number, second)``` where ```def contains```:
1. returns ```true``` if ```second``` contains ```number```
2. ```false``` if doesnt
3. not uses ```not in``` but regular ```for``` loop

File: Array Union/1_Array_Union.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			delete_element(first,number)
+		elif number in second:
+			delete_element(second,number)
+		return []
+def delete_element(array,element):
+	array.remove(element)
+	return array
+result = []
+first = (input("Введите числа:")).split(",")
+second = (input("Введите числа:")).split(",")
```
Comment (daria-gh) [inline Array Union/1_Array_Union.py]: English... :(

**Review by daria-larionova — CHANGES_REQUESTED** [review]

File: Array Union/1_Array_Union.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+		return True
+	else:
+		return False
+result = []
+first = (input("Enter numbers:")).split(",")
+second = (input("Enter numbers:")).split(",")
+while result == []:
+	result = array_union(first,second)
+if result != []:
+	print(sorted(result))
```
Comment (daria-gh) [inline Array Union/1_Array_Union.py:49]: Not distinct:

Enter numbers:1, 2, 4, 12, 2, 3, 6
Enter numbers:2, 8, 45, 12, 6, 7, 7, 7
**NOW**: [1, 2, 2, 3, 4, 6, 7, 7, 7, 8, 12, 45]
**EXPECTED**: [1, 2, 3, 4, 6, 7, 8, 12, 45]

**Review by daria-larionova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**